### PR TITLE
INT-4002: JDBC MS Doc: DB init in the beginning

### PR DESCRIPTION
**Pull request by artembilan**
JIRA https://jira.spring.io/browse/INT-4002

To avoid some confuses during developing target applications,
warn end-users that it is important to initialize data base before
running applications with JDBC Message Stores

### CODE REVIEW

File: src/reference/asciidoc/jdbc.adoc [hunk 1]
```diff
@@ -292,10 +292,19 @@ See <<jdbc-outbound-channel-adapter>> for more information about `MessagePrepare
...
 While it can be used for backing _Message Channels_ as well, you may want to consider using the `JdbcChannelMessageStore` implementation instead, as it provides a more targeted and scalable implementation.
 
+==== Initializing the Database
+
+Before starting to use JDBC Message Store components, it is important to provision target data base with the appropriate objects.
+
+Spring Integration ships with some sample scripts that can be used to initialize a database.
+In the `spring-integration-jdbc` JAR file you can find scripts in the `org.springframework.integration.jdbc` package: there is a create and a drop script example for a range of common database platforms.
+A common way to use these scripts is to reference them in a https://docs.spring.io/spring/docs/current/spring-framework-reference/html/jdbc.html#jdbc-intializing-datasource[Spring JDBC data source initializer].
+`Note that the scripts are provided as samples or specifications of the the required table and column names.
```
Comment (garyrussell) [inline src/reference/asciidoc/jdbc.adoc]: Looks like a spurious back-tick `Note - will fix on merge.

Comment (artembilan) [inline src/reference/asciidoc/jdbc.adoc]: Thanks

**Comment by garyrussell** [comment]
Merged as bb2ee967f28bb6d9625ab3fa992c5ed68e096d6b

**Comment by manueljordan** [comment]
Thanks by this update in the reference documentation.

**Pull request closed without merging**